### DISCUSSION TITLE
Move fastlane changelogs into its own component

### DISCUSSION THREAD

**Discussion by TheLastProject** [WeblateOrg/weblate#7917] [Q&A] [answered]
I run [Catima](https://hosted.weblate.org/projects/catima/) on Weblate and right now the Fastlane component is kinda... messy.

It has about 5 strings that really matter and are necessary for updating the metadata on Google Play and F-Droid, and hundreds on changelogs that are honestly quite low-priority.

Is there a way to move the changelogs into its own component? I figure I can probably create a new component with a File mask of `fastlane/metadata/android/*/changelogs/*`, but how would I filter them out of the main Fastlane component?

**Comment by nijel** [comment, marked as answer]
You can mark them as read only to exclude them from translation, see https://docs.weblate.org/en/latest/formats.html and https://docs.weblate.org/en/latest/admin/addons.html#addon-weblate-flags-bulk

**Reply by TheLastProject** [reply]
An useful note, but that's a bit too extreme for my usecase. I have several translators who are willing to translate changelogs which is helpful for both F-Droid and Google Play publishing, I just want to nudge translators to deprioritize changelogs over in-app strings and important Fastlane metadata.

Thanks though!

**Reply by Geeyun-JY3** [reply]
Maybe you can try using the `priority:N` flag.
See

- https://docs.weblate.org/en/latest/admin/translating.html#strings-prioritization
- https://docs.weblate.org/en/latest/admin/checks.html#customizing-behavior-using-flags

**Reply by nijel** [reply]
The changelogs automatically have lowest priority - they are offered as last strings to translate.

**Reply by TheLastProject** [reply]
Well, I suppose that is the answer I wanted. Definitely good enough :)

**Reply by nijel** [reply]
Or, to be precise, the string ordering is coded here:

https://github.com/WeblateOrg/weblate/blob/9ed9e21a05bf13e4355a4ee09be8c774ac0c59c8/weblate/formats/txt.py#L120-L133
